schema.graphql

```diff
@@ -585,3 +585,14 @@ type DeployEscrowEvent implements DeployEvent @entity {
   # event params
   minCRatio: BigDecimal!
 }
+
+type SupportedToken @entity {
```

Comment:
Do u think this is the right name? Its basiclly a one-to-many relationship between a token and oracles that have prices for it

Reply:
I did not change this. Perhaps `OracleTokens` or `RegisteredTokens`. I cannot think of anything better at the moment. I'm trying to think of a name that indicates that the token can be used in the protocol as collateral, credit, or revenue.